app/main.py

```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> Any:
+        input_args = (args, frozenset(kwargs.items()))
```

Comment:
You could limit your processing to `args` only.

Reply:
Done. Thanks for comment.